FrankiesGameTest.py

```diff
@@ -0,0 +1,28 @@
+import sys, pygame
```

Comment:
This file doesn't look relevant to your game. Generally, you don't want to push code that is unrelated to the project.